pkg/idtools/idtools.go

```diff
@@ -3,11 +3,14 @@ package idtools
 import (
 	"fmt"
 	"os"
+
+	"github.com/moby/sys/user"
 )
 
 // IDMap contains a single entry for user namespace range remapping. An array
 // of IDMap entries represents the structure that will be provided to the Linux
 // kernel for creating a user namespace.
+// Deprecated: use user.IDMap instead
```

Comment:
These need an empty line before them, otherwise they're not picked up as a deprecation comment. When updating, it's also good to make the replacement a docs-link (adding square brackets around them); those are rendered as links to the replacement on pkg.go.dev.

```suggestion

// Deprecated: use [user.IDMap] instead
```